### ISSUE TITLE
Ability to cast to chromecast devices

### ISSUE THREAD

**Issue by EricChan1989** [jeffvli/feishin#811]
### What do you want to be added?

Dear Developer,

May I know if you consider to add the capability to cast the music which is playing to the chromecast devices? It can make this application a jukebox and play music on any such audio devices.

Thanks.

### Is this a server-specific feature? (e.g. Jellyfin only)

- [ ] Yes

**Comment by molotch** [comment]
This would be a really nice feature to have.

**Comment by c-prompt** [comment]
I want to be able to cast/play on other Jellyfin devices. For example, I have a mobile phone running [Finamp](https://github.com/jmshrv/finamp) that's connected to my stereo system. I want to be able to cast to that device.

Sadly, I can't use this software without that functionality and have to stick to Jellyfin's Web app.

**Comment by urbantrout** [comment]
Feishin is great! And I too would love to see chromecast support!!

**Comment by graham2071** [comment]
As a workaround for this, you can [host your own web version of Feishin](https://github.com/jeffvli/feishin?tab=readme-ov-file#web-and-docker), so you can use any web browser supporting chromecast.

**Comment by urbantrout** [comment]
Yeah, I already do something like this. Google Chrome + Jellyfin Web. But Feishin has a better user experience for listening to music…
